modules/apigee_edge_teams/src/Form/EditTeamMemberForm.php

```diff
@@ -151,7 +154,10 @@ public function buildForm(array $form, FormStateInterface $form_state, TeamInter
   public function submitForm(array &$form, FormStateInterface $form_state) {
     $logger = $this->logger('apigee_edge_teams');
 
-    $selected_roles = array_filter($form_state->getValue('team_roles', []));
+    $selected_roles = array_filter($form_state->getValue('team_roles', []), function ($role) {
```

Comment:
Do not repeat yourself.

Reply:
A disabled checkbox would be filtered out just by `array_filter($form_state->getValue('team_roles', []));` . However,  a more throughout filter is needed in case there was some manual tampering of the form, i.e., if someone enabled the checkbox in the browser and submits it, then we need to filter it out. It's not the usual, but we can't trust user input.